test/integration/WithdrawalsAndRewards.test.ts

```diff
@@ -228,7 +228,7 @@ describe("PFP Protocol - Withdrawals and Rewards", function () {
         BigNumber.from("1500000000000000000000000")
       );
       currentPurposePrice = await pfp.getPurposePrice();
-      expect(currentPurposePrice).to.eq(10127);
+      expect(currentPurposePrice).to.eq(10637);
```

Comment:
please add [bonding-curve-scenario-test.pdf](https://github.com/Purpose-for-Profit/pfp-contracts/files/10917273/bonding-curve-scenario-test.pdf) to docs folder; I've updated it with the bonding curve price change